catalog/secrets/cache/src/main/java/org/projectnessie/catalog/secrets/cache/CachingSecretsBackend.java

```diff
@@ -51,6 +52,7 @@ public CachingSecretsBackend(SecretsCacheConfig config) {
 
     Caffeine<CacheKeyValue, Secret> cacheBuilder =
         Caffeine.newBuilder()
+            .scheduler(Scheduler.systemScheduler())
             .expireAfter(
                 new Expiry<CacheKeyValue, Secret>() {
```

Comment:
maybe switch to `Expiry.creating(...)`?

Reply:
That's a nice proposal!
It simplifies the code.